pkg/autoops/api/progressive_rollout.go

```diff
@@ -528,6 +558,23 @@ func (s *AutoOpsService) ExecuteProgressiveRollout(
 		}
 		return nil, dt.Err()
 	}
+	if errs := s.publisher.PublishMulti(ctx, []publisher.Message{event}); len(errs) > 0 {
+		s.logger.Error(
+			"Failed to publish events",
+			log.FieldsFromImcomingContext(ctx).AddFields(
+				zap.Any("errors", errs),
+				zap.String("environmentId", req.EnvironmentNamespace),
+			)...,
+		)
+		dt, err := statusInternal.WithDetails(&errdetails.LocalizedMessage{
+			Locale:  localizer.GetLocale(),
+			Message: localizer.MustLocalize(locale.InternalServerError),
+		})
+		if err != nil {
+			return nil, statusInternal.Err()
+		}
+		return nil, dt.Err()
```

Comment:
It's just a note, but because this is a background operation, we don't need to localize the error.